web-sdk/v3.md

```diff
@@ -0,0 +1,341 @@
+# Web SDK <a style="color: #111;" class="improve-docs" href="/web-sdk/v3">v3</a> <a style="margin-right: auto;" class="improve-docs" href="/web-sdk/v2">v2</a>
+
+The Fidel SDKs provide a secure way for you to add card-linking capabilities to your applications. The alternative involves using the [Fidel Cards API](https://reference.fidel.uk/reference#create-card), but you'll need to be PCI Compliant before you can use it. Using the Fidel SDKs, card details are sent directly to the Fidel API through a secure connection without exposing your servers to sensitive information. They take care of all PCI Compliance requirements, so you don't have to.
+
+**The Fidel Web SDK v3** is a JavaScript-based SDK that injects a secure HTML iframe into your web page. Your users will need to have JavaScript enabled in their browsers.
+
+The SDK provides a pre-built UI that allows you to easily collect credit card details, tokenise and link credit/debit cards with rewards services from your website, e-commerce platform or mobile web apps. However, for mobile apps, we recommend using the [Fidel Mobile SDKs](/mobile-SDKs).
+
+You can customise the UI of the SDK to match your use case. All modern desktop and mobile browsers are supported, including Chrome, Firefox, Safari, Microsoft IE and Edge. Please contact us at [devrel@fidel.uk](mailto:devrel@fidel.uk) if you experience any browser-related issues.
```

Comment:
@jordrake I know this text is already present on our current docs, but are we sure we support IE? Should we say IE+10 or something instead?

Reply:
Talked to Jordan, made it IE11.